packages/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-settings-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingSettingsPlugin.kt

```diff
@@ -25,21 +25,23 @@ open class ExpoAutolinkingSettingsPlugin : Plugin<Settings> {
 
       // Includes the `expo-gradle-plugin` subproject.
       settings.includeBuild(
-        expoGradlePluginPath
+        expoGradlePluginsFile.absolutePath
       )
     }
   }
 
-  private fun getExpoGradlePluginPath(settings: Settings): File {
+  private fun getExpoGradlePluginsFile(settings: Settings): File {
     val expoModulesAutolinkingPath =
       settings.providers.exec { env ->
         env.workingDir(settings.rootDir)
         env.commandLine("node", "--print", "require.resolve('expo-modules-autolinking/package.json', { paths: [require.resolve('expo/package.json')] })")
       }.standardOutput.asText.get().trim()
 
+    val expoAutolinkingDir = File(expoModulesAutolinkingPath).parentFile
+
     return File(
-      expoModulesAutolinkingPath,
-      "../android/expo-gradle-plugin"
+      expoAutolinkingDir,
+      "android/expo-gradle-plugin"
     )
```

Comment:
Fun fact:
Checking if the path exists when it contains "/../" doesn't always work 🤷 
So I've decided to use `parentFile` instead.